src/Stryker.Core/Stryker.Core/Mutants/MutantPlacer.cs

```diff
@@ -15,6 +15,12 @@ public static class MutantPlacer
 
         public static IEnumerable<string> MutationMarkers => new[] { MutationConditional, MutationIf };
 
+        public static BlockSyntax PlaceContextMarker(BlockSyntax block)
```

Comment:
This places the Static context marker right? Perhaps it should be PlaceStaticMarker or PlaceStaticContextMarker

Reply:
In my mind this marker could be used in several places, but this is too early to decide on a design
==> renamed